src/test.sol

```diff
@@ -28,7 +28,7 @@ contract DSTest {
     bool public IS_TEST;
     bool public failed;
 
-    constructor() internal {
+    constructor() {
```

Comment:
I think it needs to be explicitly `public` if we're still to be compatible with whatever made us put `internal` there in the first place.

Reply:
I'm making a bigger pr with some other additions, so can address it there. Also have a script for recursive dappsys updating :)